doc/release/1.4.0-notes.rst

```diff
@@ -126,14 +368,620 @@ are all equally likely if ``loguniform(10 ** 0, 10 ** 2).rvs()`` is used.
 
 Other changes
 =============
+The ``LSODA`` method of `scipy.integrate.solve_ivp` now correctly detects stiff
+problems.
+
+`scipy.spatial.ckdtree` now accepts and correctly handles empty input data
+
+`scipy.stats.binned_statistic_dd` now calculates the standard deviation 
+statistic in a numerically stable way.
+
+`scipy.stats.binned_statistic_dd` now throws an error is the input data 
```

Comment:
my typo, `error is the input data` should read `error if the input data`

Reply:
Also Warren just merged into mainline the PR #10480 that is a 1.4.0 milestone with the error throw if `np.nan` or `np.inf` I think it was after this pass though. I added a corresponding note in the release notes wiki page in the corresponding section here, other changes.

Reply:
ok, made all these changes